.circleci/config.yml

```diff
@@ -465,6 +469,11 @@ workflows:
           name: mysql-itest
           extras: '[mysql,itest]'
           python_version: "3.6"
+          # Using Pandas reproduced a bug that happened when we were
+          # relying on Pandas:
+          #
+          # See https://github.com/bluelabsio/records-mover/pull/152
+          pandas_version: ">1"
```

Comment:
Reproduced here: https://app.circleci.com/pipelines/github/bluelabsio/records-mover/1812/workflows/c7b38b71-8ef9-4f6c-b096-10fe82d0c2ca/jobs/15504

Reply:
Is there any sense in running these tests with old pandas as well? I mean, hopefully people aren't really using old pandas much anymore (should that maybe become the default pandas version unless overridden?) I guess that's a lot of testing to do for a relatively small segment of functionality

Reply:
Honestly I'm only halfway convinced it makes sense to keep this in the tests as a special case - this one line of code is currently the issue, there's nothing else special about MySQL and Pandas, and now we have a unit test for this issue, which is lower on the test pyramid (and we run unit tests on both Pandas <1 and Pandas >1).